host/syscall_u_wrapper.c

```diff
@@ -16,6 +16,9 @@
 /* Override oe_call_enclave_function() calls with _call_enclave_function(). */
 #define oe_call_enclave_function _call_enclave_function
 
+// Obscure the generated creation function by renaming it.
```

Comment:
nit: can we pick one comment style in this file and stick with it?  I.e., line 16, line 19, and line 22 currently alternate styles.

Reply:
Done.